README.md

```diff
@@ -254,14 +254,6 @@ and enable [core library desugaring](https://developer.android.com/studio/write/
 
 ### Gradle
 
-- Add the bintray repository:
-
-```kotlin
-repositories {
-    maven(url = "https://kotlin.bintray.com/kotlinx/") // soon will be just jcenter()
-}
```

Comment:
In Gradle builds, Maven Central repository has to be defined explicitly as any other respository. So it makes sense to note "Add `mavenCentral()` to `repositories` if it's not already there, and give an example.

Reply:
Fixed.